app/timetables/admin.py

```diff
@@ -1,11 +1,11 @@
 from django.contrib import admin
 
-from .models import Course, Meal, MealOption, Weekday, Timetable, Dish
+from . import models
 
```

Comment:
@andela-kerinoso Nice one here with the import. What do you think about having the extra line spacing in line with our usual format?